### DISCUSSION TITLE
How to get detection patterns' position in QRcode?

### DISCUSSION THREAD

**Discussion by shane-S** [journeyapps/zxing-android-embedded#597] [Q&A]
I want to know the distance between 2 detection pattern(left top and left bottom), but don't know how to get the position data of them.

**Comment by rkistner** [comment]
The closest to what is currently supported is the `possibleResultPoints` callback:

https://github.com/journeyapps/zxing-android-embedded/blob/13440ad8751dcaba4177fa50dfee639dac52a53d/zxing-android-embedded/src/com/journeyapps/barcodescanner/BarcodeCallback.java#L29

For QR codes, this is potential detection patterns, but there may be many false results. This is typically just used to give some feedback to the user while scanning.

You can see the implementation here:
https://github.com/zxing/zxing/blob/master/core/src/main/java/com/google/zxing/qrcode/detector/Detector.java
